leo_filters/src/imu_filter_parameters.yaml

```
@@ -42,4 +42,31 @@ imu_filter:
     read_only: true
     validation:
       bounds<>: [1, 600]
-  
+  angular_velocity_threshold:
+    type: double
+    default_value: 0.2
+    description: "Threshold for the angular velocity to determine if the filter is in steady state."
+    read_only: false
+    validation:
+      gt<>: [0.0]
+  acceleration_threshold:
+    type: double
+    default_value: 0.1
+    description: "Threshold for the acceleration to determine if the filter is in steady state."
+    read_only: false
+    validation:
+      gt<>: [0.0]
+  delta_angular_velocity_threshold:
+    type: double
+    default_value: 0.01
+    description: "Threshold for the change in angular velocity to determine if the filter is in steady state."
+    read_only: false
+    validation:
+      gt<>: [0.0]
+  required_steady_time:
+    type: double
+    default_value: 1.0
+    description: "Time required to be in steady state before bias estimation is performed (in seconds)."
```

IMO it makes more sense if we don't consider the IMU to be in steady state before the required time has passed. My proposal:
```suggestion
    description: "How long should the IMU reading meet the steady thresholds before we determine it's in steady state (in seconds)."
```